sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/CassandraResourcesOperationsTests.cs

```diff
@@ -33,6 +32,7 @@ public async Task ClearAndInitialize()
             {
                 InitializeClients();
                 resourceGroupName = Recording.GenerateAssetName(CosmosDBTestUtilities.ResourceGroupPrefix);
+                databaseAccountName = Recording.GenerateAssetName("amecassandratest");
```

Comment:
Entropy of this is still low, but we will change this soon anyway

Reply:
I will update all random name generations in CosmosDB after I finished the random generator in the test framework.